### PR TITLE
Add chain frenchain

### PR DESCRIPTION
**Pull request by shopglobal**
Testnet:
chainID: 444
networkID: 444

Mainnet:
chainID: 4444
networkID: 4444

Thanks,M

### CODE REVIEW

**Comment by ligi** [comment]
please ping me after launch - then I will check the pr

**Comment by shopglobal** [comment]
I misspoke, the text was a copypasta from last PR. 
Frenchain project is launched. 

https://github.com/fren-chain/go-frenChain

**Comment by shopglobal** [comment]
Just merged the networkID swap https://github.com/fren-chain/go-frenChain/pull/1

File: _data/chains/eip155-4444.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Frenchain",
+  "chain": "fren",
+  "rpc": ["https://rpc-01.frenchain.app"],
```
Comment (ligi) [inline _data/chains/eip155-4444.json:4]: this one returns the testnet chainID 444

File: _data/chains/eip155-4444.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "infoURL": "https://frenchain.app",
+  "shortName": "FREN",
+  "chainId": 4444,
+  "networkId": 4444,
+  "icon": "fren",
+  "explorers": [
+    {
+      "name": "blockscout",
+      "url": "https://mainnet.frenscan.io",
```
Comment (ligi) [inline _data/chains/eip155-4444.json:19]: does not work

**Review by ligi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**